### ISSUE TITLE
Real-time Demo

### ISSUE THREAD

**Issue by jeffryuiop** [SoftwareGift/FeatherNets_Face-Anti-spoofing-Attack-Detection-Challenge-CVPR2019#60]
In your demo (https://www.youtube.com/watch?v=giznA4WU2-4), which model did you use? Was it the ensemble, or was it only FeatherNetB?

**Comment by SoftwareGift** [comment]
> In your demo (https://www.youtube.com/watch?v=giznA4WU2-4), which model did you use? Was it the ensemble, or was it only FeatherNetB?

Only FeatherNetB

**Comment by jeffryuiop** [comment]
okay thank you for the reply.
I tried to make a demo using your pretrained weight, but the result is not as good as yours (youtube video).
To be honest I'm still confused with the inference process that you stated in other forum.  It would be great if you can clarify a few things.

1. So the output of the model is 1024 vector, but you only use the first 2 to put into softmax in order to make the inference?
2. Did you also use FeatherNetB for IR in the demo?
